cluster/config-defaults.yaml

```diff
@@ -809,3 +809,6 @@ hyped_article_lifecycle_management: "false"
 
 # enable SizeMemoryBackedVolumes feature flag
 enable_size_memory_backed_volumes: "true"
+
+# specify hostname max allowed length
+hostname_max_length: 57
```

Comment:
I'm a bit surprised CLM doesn't complain. Config Items are always strings so it should ideally be `"57"` defined here.

Reply:
interesting, will fix it.

Reply:
https://github.com/zalando-incubator/kubernetes-on-aws/pull/5468